proparse/src/main/java/org/prorefactor/proparse/antlr4/Lexer.java

```diff
@@ -541,7 +560,8 @@ ProToken digitStart() throws IOException {
   }
 
   ProToken plusMinusStart(int inputType) throws IOException {
-    int ttype = ProParserTokenTypes.NUMBER;
+	  LOGGER.trace("Entering plusMinusStart()");
+	  int ttype = PreprocessorParser.NUMBER;
     for_loop : for (;;) {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png 'Severity: MAJOR') Refactor the code to remove this label and the need for it. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonar.riverside-software.fr/coding_rules#rule_key=squid%3ALabelsShouldNotBeUsedCheck)